### ISSUE TITLE
Combine with R/Pharma tutorials and open up for contributions?

### ISSUE THREAD

**Issue by rossfarrugia** [pharmaverse/end_to_end_examples#28]
Any way we can combine our examples site with https://github.com/rinpharma/r-for-clinical-workflows and examples like https://github.com/rinpharma/r-for-clinical-workflows/blob/main/tutorials/creating-adam-adsl-part1.qmd that might be a bit easier for others to contribute to.

@epijim will discuss next week with you if catch you in Basel, as @statasaurus raised this today from a discussion with Posit.

**Comment by bms63** [comment]
Should we proceed with making the ADPC example?  We were going to follow similar structure to ADSL.  No issues on where it lands for hosting.

**Comment by rossfarrugia** [comment]
Yes sure go ahead for now, and then wherever we agree these will live we can transfer over later. The renv.lock file does have some issues in this repo heads up so PRs might fail CI checks for now. We have an open PR facing this issue currently but not putting time into fixing until we know for sure whether we'll move to the Posit repo.

**Comment by bms63** [comment]
cool good to know.
